envs/dev/templates/team-burnham.yaml

```diff
@@ -3,6 +3,8 @@ kind: Application
 metadata:
   name: team-burnham
   namespace: argocd
+  labels:
+    env: {{ .Values.spec.env }}
```

Comment:
hmm, what if we do like `values` and do
```yaml
  labels: |
        {{- toYaml .Values.labels | nindent 6 }}
```

Reply:
sounds improvements :)

Reply:
in fact I remove them, the labels are not applied on the Application object